target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr840n-v2.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *  TP-LINK TL-WR841N/ND v9/v11 / TL-WR842N/ND v3
```

Comment:
OK, I'm preparing changes, but are you really sure?
It would add some code (yes, only a few bytes) to the firmware, which isn't necessary...